matlab/test/arrow/tabular/tRecordBatch.m

```diff
@@ -223,6 +223,169 @@ function SchemaNoSetter(tc)
                 "MATLAB:class:SetProhibited");
         end
 
+        function GetColumnByName(testCase)
+            % Verify that columns can be accessed using a field name.
```

Comment:
can be accessed using a field name -> can be accessed by name

Reply:
Fixed.